pkgs/applications/networking/instant-messengers/fluffychat/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitLab
+, flutter
+, olm
+}:
+
+flutter.mkFlutterApp rec {
+  pname = "fluffychat";
+  # 42.3 is latest
```

```suggestion
  # 0.42.3 is latest
```
and the latest is already 1.0.1 https://gitlab.com/famedly/fluffychat/-/tags

---

Hrm I should try the latest version then